test/python/compiler/test_assembler.py

```diff
@@ -110,6 +114,28 @@ def test_assemble_opaque_inst(self):
         self.assertEqual(qobj.experiments[0].instructions[0].memory, [3, 0])
         self.assertEqual(qobj.experiments[0].instructions[0].params, [0.5, 0.4])
 
+    def test_assemble_single_schedule(self):
```

Comment:
We will have to add many more test schedules for of edge cases that may be triggered.